src/AltinnCore/Runtime/Controllers/InstanceController.cs

```diff
@@ -390,7 +382,6 @@ public async Task<IActionResult> StartService(StartServiceModel startServiceMode
                }).ToList();
 
             HttpContext.Response.Cookies.Append("altinncorereportee", startServiceModel.ReporteeID.ToString());
-            _logger.LogInformation($" // 404 // Setting startService Model view: {startServiceModel} ");
```

Comment:
probably the logging must be left for a while, so that it is helpful to analyse when it fails

Reply:
Okok, I'll close the PR then, and we can reopen at a later time